### PR TITLE
Version 2.0.0

### PR DESCRIPTION
**Pull request by elpete**
Update docs for version 2.0.0

### CODE REVIEW

**Comment by elpete** [comment]
@lmajano Are we waiting on anything to announce ColdBox Elixir 2.0?

**Comment by lmajano** [comment]
Sorry Eric been traveling. 
Do you have acces to the Ortus blog already?

File: configuration_options.md [hunk 1]
```diff
@@ -434,6 +394,3 @@ You can override or add configuration options in your `Gulpfile.js` by referenci
 elixir.config.production = true;
 ```
 
-# Custom Configuration File
```
Comment (lmajano) [inline configuration_options.md]: Was this removed in the new elixir?

Comment (elpete) [inline configuration_options.md]: This was removed in the new version of Elixir.  We could work to put it back if we wanted.  Not too much work, but also not sure how much benefit.

File: configuration_options.md [hunk 2]
```diff
@@ -3,426 +3,386 @@
...
+         |
+         | Very likely, you will never need/want to modify this property.
+         | However, for the instances where your app's views directory
+         | is located in a different spot, please modify as needed.
+         |
+         */
+
+        viewPaths: {
+            "views"   : "views",
+            "layouts" : "layouts"
```
Comment (elpete) [inline configuration_options.md]: Don't think so.  I'll take it out.

**Pull request merged**